src/main/java/com/fcgl/madrid/forum/model/CommentRequest.java

```diff
@@ -0,0 +1,45 @@
+package com.fcgl.madrid.forum.model;
+
+import com.fcgl.madrid.forum.dataModel.Post;
+
+public class CommentRequest {
+    private String message;
+    private Integer userId;
+    private Post post;
+
+    /**
+     * Request Parameters for post
+     * @param message
+     * @param userId
+     * @param post
+     */
+    public CommentRequest(String message, Integer userId, Post post){
```

Comment:
Typically we would want to validate that the `postId` is a valid one (that ID is mapped to an actual post) since we have access to the database. But to keep latency low I won't require it. I thought that the foreign key constraint would not allow this but it looks like JPA set the foreign key relation to `MATCH SIMPLE` which doesn't check if foreign key is null or if it exists. 

The only way the `postId` would be invalid is if our frontend code is incorrect, or if our getRequest for posts has a bug, or if someone got access to our API (but hopefully our authentication/authorization implementation will take care of that) 

curl -X POST -d userId=20 -d post=10000 -d message=testing http://localhost:8082/forum/comment/v1/postComment

Reply:
Got it to work.  first image is if the id dont exist second is when the postid does exist
![image](https://user-images.githubusercontent.com/9799136/63650054-69500200-c746-11e9-879b-36b5904be769.png)
![image](https://user-images.githubusercontent.com/9799136/63650084-fa26dd80-c746-11e9-8418-6c3d52c4a0b4.png)